movies-search/src/MovieDetail/MovieDetailsContainer.jsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { getMovieById } from '../actions.jsx'
+import { connect } from 'react-redux';
+import MovieDetails from './MovieDetails';
+
+class MovieDetailsContainer extends React.Component {
+    componentDidMount() {
+        if(this.props.match.params.number) {
+            this.props.fetchDataDetails(parseInt(this.props.match.params.number, 10))
+        }
+    }
+
+    render() {
+        return <div>
+            <MovieDetails movieDetails={this.props.movieDetails} />
+        </div>;
+    }
+}
+
+const mapDispatchToProps = (dispatch) => {
+    return {
+        fetchDataDetails: (id) => dispatch(getMovieById(id))
+    };
+};
```

Comment:
if callback function matches the signature of action creator you can simplify `mapDispatchToProps` to the following:

```javascript
const mapDispatchToProps = {
  fetchDataDetails: getMovieById
}
```